src/Compatibility/Core/src/Windows/NavigationPageRenderer.cs

```diff
@@ -710,10 +707,10 @@ async void UpdateTitleOnParents()
 
 			if (_showTitle || (render != null && render.ShowTitle))
 			{
-				if (Platform != null)
-				{
-					await Platform.UpdateToolbarItems();
-				}
+				//if (Platform != null)
+				//{
+				//	await Platform.UpdateToolbarItems();
```

Comment:
Currently none of this code is running so commenting this out doesn't break anything.